src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -77,7 +79,7 @@
     ConfigKey<Integer> REQUIRED_MEMORY = ConfigKeys.newIntegerConfigKey(
             "cloudfoundry.profile.memory", "Memory allocated for the application (MB)", 512);
 
-    @SetFromFlag("disk")
+    @SetFromFlag("disk_quota")
```

Comment:
tricky, I think brooklyn convention is to use camelCase but it is a good point to keep coherence with cf manifest. Not sure, leave it as disk-quota